skyportal/handlers/api/followup_request.py

```diff
@@ -954,10 +958,12 @@ def observation_schedule(
         The planned observations associated with the request
     instrument : skyportal.models.instrument.Instrument
         The instrument that the request is made based on
-    observation_start: astropy.time.Time
+    observation_start : astropy.time.Time
         Start time for the observations
-    observation_end: astropy.time.Time
+    observation_end : astropy.time.Time
         End time for the observations
+    standards : pandas.DataFrame
+        Standard stars for inclusion in the observation plan.
```

Comment:
What needs to be the columns of this table?

Reply:
good idea.